rsts/howto/sandbox.rst

```diff
@@ -21,8 +21,95 @@ Flyte can be run using a Kubernetes cluster only. This installs all the dependen
+            #. To debug you can try a simple excercise - run nginx as follows::
+
+                docker run -it --rm -p 8083:80 nginx
+
+               Now connect to `locahost:8083 <localhost:8083>`__. If this does not work, then for sure the networking is impacted, please restart docker daemon.
+
+    .. tab:: Using Docker for Mac
+
+        #. `Install Docker for mac with Kubernetes as explained here <https://www.docker.com/blog/docker-mac-kubernetes/>`_
+        #. Make sure Kubernetes is started and once started make sure your kubectx is set to the `docker-desktop` cluster, typically ::
+
+                kubectl config set-context docker-desktop
+
+        #. Install Flyte::
+
+            kubectl create -f https://raw.githubusercontent.com/flyteorg/flyte/master/deployment/sandbox/flyte_generated.yaml
+
+
+        #. Connect to `FlyteConsole <localhost/console>`__
+
+    .. tab::  Using Minikube (Not recommended)
+
+        #. Install `Minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_
+
+        #. Install Flyte::
+
+            kubectl create -f https://raw.githubusercontent.com/flyteorg/flyte/master/deployment/sandbox/flyte_generated.yaml
+
+
+        .. note::
+
+            - Minikube runs in a Virtual Machine on your host
+            - So if you try to access the flyte console on localhost, that will not work, because the Virtual Machine has a different IP address.
+            - Flyte runs within Kubernetes (minikube), thus to access FlyteConsole, you cannot just use https://localhost:30081/console, you need to use the IP address of the minikube VM instead of localhost
+            - Refer to https://kubernetes.io/docs/tutorials/hello-minikube/ to understand how to access a
+                also to register workflows, tasks etc or use the CLI to query Flyte service, you have to use the IP address.
+            - If you are building an image locally and want to execute on Minikube hosted Flyte environment, please push the image to docker registry running on the Minikube VM.
+            - Another alternative is to change the docker host, to build the docker image on the Minikube hosted docker daemon. https://minikube.sigs.k8s.io/docs/handbook/pushing/ provides more detailed information about this process. As a TL;DR, Flyte can only run images that are accessible to Kubernetes. To make an image accessible, you could either push it to a remote registry or to a regisry that is available to Kuberentes. In case on minikube this registry is the one thats running on the VM.
+
+    .. tab:: Install Flyte in EKS/GKE etc
+
+        Refer to the :ref:`howto-sandbox-dedicated-k8s-cluster` guide.
+
+        Once you've deployed flyte sandbox to a cloud provider, read on below to run your first workflow remotely.
+
 
 
 .. _howto-sandbox-dedicated-k8s-cluster:
```

Comment:
Do we still need this section here?

Reply:
I think there will be folks that will invariably try this out